### PR TITLE
Remove ~ that people keep putting in the system variable

### PR DESCRIPTION
**Pull request by JMM889901**
if this actually needed, just close this, I have heard mixed things about including the ~ not working

### CODE REVIEW

**Comment by JMM889901** [comment]
womp

**Comment by GeckoEidechse** [comment]
I don't have an affected CPU to test this. If anyone with an affected CPU could test this and report back whether the change is correct so that I can merge or close, that'd be great ^^

**Comment by itscynxx** [comment]
i've hopefully got someone (righthonorable on the Discord server) testing this over the weekend, but if anyone else wants to do it sooner feel free to

**Comment by LumiKattlin** [comment]
Tested on my pc with a 12th gen and without the ~ it worked for me

File: docs/installing-northstar/troubleshooting.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ On newer Intel CPUs you might see a message like this
 If you are seeing this in the main menu of TF|2 and have a 10th or 11th generation Intel CPU this is a bug which has a simple fix:
 
 In the Windows Start menu on the bottom left search for "Edit the system environment variables" and open the program. In the "advanced" tab click on "Environment Variables..." near the bottom.\
-In System Variables (not user variables) click "New..." and add a new system variable where the variable name is `OPENSSL_ia32cap` and the value is `~0x200000200000000`. Make sure to click OK to apply the changes. Finally restart your device and you should be good to go.
+In System Variables (not user variables) click "New..." and add a new system variable where the variable name is `OPENSSL_ia32cap` and the value is `0x200000200000000`. Make sure to click OK to apply the changes. Finally restart your device and you should be good to go.
 
 If you're on Linux, you can set the appropriate environment variable via `env OPENSSL_ia32cap=~0x20000000 %command%`.
```
Comment (itscynxx) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
If you're on Linux, you can set the appropriate environment variable via `env OPENSSL_ia32cap=0x20000000 %command%`.
```
Should the Linux equivalent not also be changed?

**Review by itscynxx — COMMENTED** [review]
Unless Linux works differently than this

File: docs/installing-northstar/troubleshooting.md (hunk 1 above)
Comment (JMM889901) [reply docs/installing-northstar/troubleshooting.md]: I'll be honest I just didn't notice that part, I also have no idea if it matters for Linux, I'm going to assume so so I'll commit this

**Comment by itscynxx** [comment]
Requested changes have been made, and the guide has been tested to work without the `~` in it

**Comment by GeckoEidechse** [comment]
Wait, so both the version with and without the `~` work? Or just the one without?

**Comment by itscynxx** [comment]
> Wait, so both the version with and without the `~` work? Or just the one without?

Lumi tested it and says it worked, I've recommended it to a few users and it's worked for them as well

Additionally, I'll have a 12th gen I'll put together tonight instead of waiting. Can test myself later as long as I don't skill issue and break something

**Comment by itscynxx** [comment]
Just tested.

On Northstar, I didn't need to set anything to get in game. Installed my i7 12700kf yesterday, didn't touch variables, and was able to connect normally

On vanilla I did have the issue tho. Worked perfectly fine without the `~`

**Comment by pg9182** [comment]
For the record, `~0x20000000` clears the 29th bit of the result from cpuid, while `0x20000000` replaces the result with one with just the 29th bit set ([OPENSSL_ia32cap.3](https://www.openssl.org/docs/manmaster/man3/OPENSSL_ia32cap.html)). The 29th bit [represents](https://sourcegraph.com/github.com/torvalds/linux@95c8a35f1c017327eab3b6a2ff5c04255737c856/-/blob/arch/x86/include/asm/cpufeatures.h?L270) the Intel SHA Extensions.

As such, if the second one (without the `~`) works, the actual solution has nothing to do with the 29th bit; it's one of the others.

**Comment by itscynxx** [comment]
Should be noted that this works more often than with the tilda
Another example of removing the tilda fixing the issue (whereas with the tilda, it didnt fix the connecting issue): https://discord.com/channels/920776187884732556/1102373220121845880/1210782379304689664

**Comment by GeckoEidechse** [comment]
Another example of removing `~` fixing it

https://discord.com/channels/920776187884732556/942505193893945394/1238545834791145502

**Comment by GeckoEidechse** [comment]
Another example of removing `~` fixing it

https://discord.com/channels/920776187884732556/1259250442500702252/1259256019616399400

**Pull request merged**